lupa/_lupa.pyx

```diff
@@ -242,12 +248,24 @@ cdef class LuaRuntime:
     cdef object _attribute_getter
     cdef object _attribute_setter
     cdef bint _unpack_returned_tuples
+    cdef size_t _max_memory
+    cdef size_t* _memory_left
```

Comment:
Why not use a plain `size_t` integer attribute here and take the address of that, instead of allocating memory and storing the pointer to that?